### PR TITLE
Fix delete command data validation and output message

### LINKED ISSUE #153: [Alek] `delete contact` description use "person" to refer to contact list
- I guess technically this is invalid format, since you already specify index must be a positive number 
- perhaps you can change "person" to "contact" for consistency
image

### PR DESCRIPTION
**Pull request by AlekKwek18**
- Fixes #153

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/DeleteCommandParser.java [hunk 1]
```diff
@@ -28,8 +30,14 @@ public DeleteCommand<?> parse(String args) throws ParseException {
...
         String entity = ParserUtil.parseEntity(entityString);
-        Index index = ParserUtil.parseIndex(indexString);
 
+        Index index;
+
+        try {
+            index = ParserUtil.parseIndex(indexString);
+        } catch (ParseException e) {
+            throw new ParseException(MESSAGE_INVALID_INDEX);
+        }
```
Comment (KengHian) [inline src/main/java/seedu/address/logic/parser/DeleteCommandParser.java:40]: Do not use this exception.  

parseIndex already validate the index and will throw exception if necessary.

**Review by KengHian — APPROVED** [review]
Overall LGTM.

File: src/main/java/seedu/address/logic/parser/DeleteCommandParser.java (hunk 1 above)
Comment (AlekKwek18) [reply src/main/java/seedu/address/logic/parser/DeleteCommandParser.java]: Roger, will remove this check

**Review by Green-Tea-123 — APPROVED** [review]
LGTM.

**Pull request merged**